core/templates/domain/improvements/improvements.constants.ts

```diff
@@ -16,52 +16,53 @@
  * @fileoverview Constants related to improvements tasks.
  */
 
-const constants = require('constants.ts');
+import { AppConstants } from 'app.constants';
 
-export namespace ImprovementsConstants {
```

Comment:
Please do not change the export type of this, it was intentionally designed to be exported as a namespace.

Reply:
Actually we've decided a new structure for constants i.e.

```typescript
export const SomeConstants {
  SOME_CONST: 'some value'
} as const;
```

So, also changed this file. Can you please explain why `namespace` is needed here.

Reply:
I see, I expected namespaces to be required for exporting "enum strings" to be used in discriminated unions. Since everything compiles even after using the new syntax, I suppose it's fine after all.

Never mind, thanks!